lib/engine/game/g_18_oe/game.rb

```diff
@@ -267,6 +501,13 @@ def after_par(corporation)
           corporation.spend(TRACK_RIGHTS_COST[region], @bank)
         end
 
+        def issuable_shares(entity)
+          return [] if !entity.corporation? || (entity.corporation? && entity.type != :major)
```

Comment:
```
    return [] if !entity.corporation? || entity.type != :major
```
Testing `entity.corporation?` a second time is redundant, it will not be evaluated unless it's already been found to be true.